distribution/kernel/carbon-home/repository/resources/conf/default.json

```diff
@@ -22,7 +22,7 @@
   "realm_manager.data_source": "SHARED_DB",
   "realm_manager.properties.isCascadeDeleteEnabled": true,
   "user_store.properties.ReadGroups": true,
-  "user_store.properties.UsernameJavaScriptRegEx": "^[\\S]{3,30}$",
+  "user_store.properties.UsernameJavaScriptRegEx": "^[^\\s*?%]{3,30}$",
```

Comment:
This regex allows the spaces in the middle of the word. i.e "I ndeewari"

Reply:
`[^\\s*?%]` is a negated character class, denoted by [^...], which means it matches any character that is not (^) a whitespace (\s), an asterisk (*), a question mark (?), or a percentage sign (%)

Tested in the IS pack and it is also not allowing this username.
<img width="456" alt="image" src="https://github.com/wso2/carbon-kernel/assets/75057725/ed34d416-9ab0-435a-94b1-2983e9b51374">